README.md

```diff
@@ -342,4 +342,6 @@ wsa://com.android.settings
 | Zenly (w/o GMS) | 4.55.2 | ⚠️ | App crashes after login, but background location works |
 | Zoom | 5.8.3.2634 | ⚠️ | Camera severely glitched, share screen doesn't work due to WSA's windowed nature. | |
 | СберБанк (SberBank) | 12.9.0 | ✅ ||
-| Rocket League Sideswipe | 1.0 (356721) | ❌ | OpenGL ES 3.1 unsupported |
+| Rocket League Sideswipe | 1.0 (356721) | ❌ | OpenGL ES 3.1 unsupported ||
+| Real Racing 3 | 10.1.0 | ✅ |no keyboard support, controller works. ||
+| google play games | current | ✅ |works. |
```

Comment:
Not sure what this line means. Is this an actual app name?

Reply:
If you are asking about Google Play Games, yes that is an actual app name. 

[Google Play Games - Apps on Google Play](https://play.google.com/store/apps/details?id=com.google.android.play.games)

Reply:
Ah, funny. We'll need a version number though @brhv098 or delete this.